pages/00-background.md

```diff
@@ -23,29 +21,29 @@ The browser community, commercial and non-profit trust store owners, and Standar
 - Requiring increasingly shorter lifetimes of the validity of certificates used for publicly trusted web services
 - Public disclosure of all certificate policies, practices and procedures by all Certification Authorities
 
-This Certificate Policy was developed for Federal Government missions and attempts to address Internet PKI requirements defined and governed by six (6) communities.
+This Certificate Policy was developed for Federal Government missions and addresses Internet PKI requirements defined and governed by six (6) communities.
 
+- Federal Government security requirements, standards and missions
 - [CA Browser Forum](https://cabforum.org/){:target="_blank"}
 - [Microsoft Trust Store Program](https://technet.microsoft.com/en-us/library/cc751157.aspx){:target="_blank"}
 - [Apple Trust Store Program](https://www.apple.com/certificateauthority/ca_program.html){:target="_blank"}
 - [Mozilla Community and Trust Store Program](https://www.mozilla.org/en-US/about/governance/policies/security-group/certs/){:target="_blank"}
 - [Google Chromium](https://www.chromium.org/Home/chromium-security/certificate-transparency){:target="_blank"}
-- Federal Government security requirements, standards and missions
 
 This Certificate Policy was scoped to encompass:
 
 - _Organizational Validation_ and _Domain Validation_ certificates for domains
-- .mil and .gov web services intended to be trusted by the public
+- .mil and .gov web services intended to be trusted by consumers and partners on the public Internet
```

Comment:
Happy to dicker over this some other time/place, but I think it's more important to emphasize the external nature of the users/devices, than to emphasize the internet as the network which will be serving these things. In addition to some inevitable and appropriate use of these certificates for "intranet" services, I think "consumers and partners" might trivialize the size/scope of the audience.

Suggestion:

> ...trusted by public visitors, consumers, and partners